src/components/budget/dto/create-budget.dto.ts

```diff
@@ -1,25 +1,29 @@
 import { Type } from 'class-transformer';
 import { MinLength, ValidateNested } from 'class-validator';
-import { Field, InputType, ObjectType } from 'type-graphql';
+import { Field, ID, InputType, ObjectType } from 'type-graphql';
 import { Budget, BudgetStatus } from '../budget';
 
 @InputType()
 export abstract class CreateBudget {
   @Field({ nullable: true })
   @MinLength(2)
   readonly status: BudgetStatus;
```

Comment:
I think status is just Pending to start with, no need to specify here

Reply:
that's fine